_maps/shuttles/starfury_corvette.dmm

```diff
@@ -120,6 +120,7 @@
 "ao" = (
 /obj/machinery/light/small/directional/west{
 	brightness = 3;
+	
 	},
```

Comment:
No

Reply:
this just happened on its own when i ran updatepaths i have no clue why